eq-author/src/App/page/Preview/CalculatedSummaryPreview.test.js

```diff
@@ -14,6 +15,27 @@ describe("CalculatedSummaryPreview", () => {
 
   actSilenceWarning();
 
+  const commentsSubscription = gql`
+    subscription CommentsUpdated($pageId: ID!) {
```

Comment:
You could import the query from this file `eq-author/src/components/CommentsPanel/commentSubscription.graphql`